### ISSUE TITLE
Dark theme

### ISSUE THREAD

**Issue by marzojr** [exult/exult-web#12]
Recently, I added a dark theme for Seven Towers. I am thinking of doing the same for the main Exult web site.

The catch is that the way I am doing it will make the site use the dark theme automatically for anyone that has their system/browser set to prefer dark mode; specifically, by adding

```css
@media screen and (prefers-color-scheme: dark) {
    /* dark theme colors go here */
}
```

to the CSS file, the site will auto-switch to dark mode if the user prefers it.

Any objections against this? Or any suggestions of color theme?

**Comment by DominusExult** [comment]
Not really an issue with this, but wondering how the dark theme will look :)

**Comment by marzojr** [comment]
Fixed in a2a4bc73c845fafb09b4a78ef7a90a592fdbb460
